src/sync.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
-rsync -avzWS $IPT_DATA_DIR /root/s3data/ --delete-after --no-compress
-echo $(date) "Backup complete" >> /var/log/backup.log
-echo $(date) "Backup complete"
+
+echo "$(date -I) Sync starting" >> /var/log/sync.log
+# Sync from local to remote and if sucessful, remote to local (only newer files)
+rclone sync --update --delete-during --verbose --transfers 10 --checkers 10 --contimeout 60s --timeout 300s --retries 3 --low-level-retries 10 --exclude .~tmp~/ $IPT_DATA_DIR sigma2:$S3_BUCKET_NAME && \
+rclone sync --update --verbose --transfers 10 --checkers 10 --contimeout 60s --timeout 300s --retries 3 --low-level-retries 10 --exclude .~tmp~/ sigma2:$S3_BUCKET_NAME $IPT_DATA_DIR && \
```

Comment:
You didn't add the --delete-during flag because we never want to delete stuff in the IPT_DATA_DIR right? Corema files should never get deleted anyway... I think this is ok